nostalgia/sources/facebook/posts.py

```diff
@@ -1,9 +1,8 @@
 import re
-import just
```

Comment:
Were all these refactorings picked up by some sort of linter?  Or done by hand?

Reply:
Just optimise imports by intellij idea